webpack/assets/javascripts/react_app/components/ModelsTable/ModelsTable.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Table } from '../common/table';
+import { STATUS } from '../../constants';
+import MessageBox from '../common/MessageBox';
+import { translate as __ } from '../../common/I18n';
+import createModelsTableSchema from './ModelsTableSchema';
+import { getURIQuery } from '../../common/helpers';
+
+class ModelsTable extends React.Component {
+  componentDidMount() {
+    this.props.getTableItems('models', getURIQuery(window.location.href));
```

Comment:
I'm also missing a spinner on initial load?

Reply:
I'm missing where you fixed this, can you give me a hint? thanks.

Reply:
No I saw you wanted it on initial load. I added a spinner every time API is being called.
I think I can fix this in one hour or less.

Reply:
Need a follow up: Pagination to be rendered inside the ModelsTable.
Right now it is rendered from `erb`. I am checking how much work it required. If it is not a lot, I will update this commit.